src/ds/d400/d400-device.cpp

```diff
@@ -964,7 +964,12 @@ namespace librealsense
         register_info(RS2_CAMERA_INFO_DFU_DEVICE_PATH, group.uvc_devices.front().dfu_device_path);
 
         if (usb_modality)
+        {
+            register_info(RS2_CAMERA_INFO_CONNECTION_TYPE, "USB");
             register_info(RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR, usb_type_str);
+        }
+        else
+            register_info(RS2_CAMERA_INFO_CONNECTION_TYPE, "GMSL");
```

Comment:
Will this list D457 Recovery DFU PID as GMSL as well?
Or we might need to do it elsewhere?

Reply:
The behavior for D457 Recovery DFU PID will not be changed.
Today, it seems that no "USB" and no "GMSL" will appear with this device.
Do you need me to check?